lighthouse-core/test/report/html/renderer/dom-test.js

```diff
@@ -38,7 +38,7 @@ describe('DOM', () => {
       const el = dom.createElement('div');
       assert.equal(el.localName, 'div');
       assert.equal(el.className, '');
-      assert.equal(el.className, el.attributes.length);
```

Comment:
no idea what this was trying to test, so I did my best :)

Reply:
lol this is hilarious